huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_instance.go

```diff
@@ -0,0 +1,662 @@
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `Instance name.`,
+				ValidateFunc: validation.All(
+					validation.StringMatch(regexp.MustCompile(`^[A-Za-z-_0-9.]*$`),
+						"the input is invalid"),
+					validation.StringLenBetween(4, 64),
+				),
+			},
+			"engine": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Message engine. Value: reliability.`,
+			},
+			"engine_version": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Version of the message engine. Value: 4.8.0.`,
+			},
+			"total_storage_space": {
+				Type:         schema.TypeInt,
+				Required:     true,
+				ForceNew:     true,
+				Description:  `Total storage space.`,
+				ValidateFunc: validation.IntBetween(300, 3000),
+			},
+			"vpc_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `VPC ID.`,
+			},
+			"subnet_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Subnet information.`,
+			},
+			"security_group_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `ID of the security group to which the instance belongs.`,
+			},
+			"availability_zone": {
+				Type:        schema.TypeList,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Required:    true,
+				ForceNew:    true,
+				Description: `ID of the AZ where instance brokers reside and which has available resources.`,
+			},
+			"flavor_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"c6.4u8g.cluster", "c6.8u16g.cluster", "c6.12u24g.cluster", "c6.16u32g.cluster",
+				}, false),
+				Description: `RocketMQ instance specifications.`,
+			},
+			"storage_spec_code": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"dms.physical.storage.high.v2", "dms.physical.storage.ultra.v2",
+				}, false),
+				Description: `Storage I/O specification.`,
+			},
+			"description": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				Description:  `Instance description.`,
+				ValidateFunc: validation.StringLenBetween(0, 1024),
+			},
+			"ssl_enable": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
+				ForceNew:    true,
+				Description: `Whether to enable SSL encryption for access.`,
+			},
+			"ipv6_enable": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     "false",
+				ForceNew:    true,
+				Description: `Whether to support IPv6.`,
+			},
+			"enable_publicip": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     "false",
```

Comment:
remove the default, the type is not correct
and if a bool type parameter defaults to false, we don't need to write default value in the code

Reply:
done